homeassistant/components/freebox/strings.json

```diff
@@ -5,6 +5,9 @@
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
           "port": "[%key:common::config_flow::data::port%]"
+        },
+        "data_description": {
+          "host": "The hostname or IP address of your Freebox router."
```

Comment:
```suggestion
          "host": "The hostname or IP address of your FRITZ!Box router."
```

Reply:
Sorry, noticed only now that there are 2 integrations in this PR.

Reply:
sorry, yes. maybe not so well chosen. Provided that the names are so close.